src/blocks/mrc_class_method_def.ts

```diff
@@ -250,6 +253,21 @@ const CLASS_METHOD_DEF = {
             }
         }
     },
+    mrcUpdateReturnInput: function (this: ClassMethodDefBlock) {
+        // Remove existing return input if it exists
+        if (this.getInput(RETURN_VALUE)) {
+            this.removeInput(RETURN_VALUE);
+        }
+        
+        // Add return input if return type is not 'None'
+        if (this.mrcReturnType && this.mrcReturnType !== 'None') {
+            this.appendValueInput(RETURN_VALUE)
+                .setAlign(Blockly.inputs.Align.RIGHT)
+                .appendField(Blockly.Msg.PROCEDURES_DEFRETURN_RETURN);
+            // Move the return input to be before the statement input
+            this.moveInputBefore(RETURN_VALUE, 'STACK');
```

Comment:
Why is the return before the statements?

<img width="707" height="221" alt="Image" src="https://github.com/user-attachments/assets/ca9c64b0-6f1f-4006-b6e5-7c1efd9e7e2f" />

I think it looks better to have the return after the statements.

<img width="695" height="179" alt="Image" src="https://github.com/user-attachments/assets/961e81ba-9e43-42a7-b264-a2f918e30642" />